tuner/tuner/common.py

```diff
@@ -272,14 +273,12 @@ class MLIRTransformation:
     embeddable: str
 
 
-def combine_tuning_specs(
-    tuner_ctx: TunerContext, td_specs: list[ir.Module]
-) -> ir.Module:
+def combine_tuning_specs(context: ir.Context, td_specs: list[ir.Module]) -> ir.Module:
```

Comment:
Why did you change this? Passing the larger context around in the tuner code seems fine to me